### ISSUE TITLE
Offline MDT ISO

### ISSUE THREAD

**Issue by elyesusa** [FriendsOfMDT/PSD#112]
I am not able to create offline ISO from PSD I am getting some errors, my work environment requires us to have an offline image just in case. Is there a workaround for this 
![Screenshot 2023-12-11 at 11 49 46 AM](https://github.com/FriendsOfMDT/PSD/assets/47729883/20c2bd62-257b-4ab7-a7a2-91db48fb8876)

**Comment by GeoSimos** [comment]
Hi @elyesusa,

Any sanitized logs to share please?

**Comment by Mase5150** [comment]
Hi @elyesusa,

Did you ever find a resolution for this issue. I would be interested to know what is was?

Thank you.

**Comment by Jamiep11** [comment]
We are getting the same results when trying to use offline media. When deploying the ISO it seems it's still looking for Litetouch.wsf. Is there a process to convert offline media backend to use new powershell scripts?

Error: Can not find script file "X:\Deploy\Scripts\LiteTouch.wsf"

<img width="355" alt="Screenshot 2024-04-08 154947" src="https://github.com/FriendsOfMDT/PSD/assets/166440728/8692ae2a-a3de-455b-bc1f-2297c2c2345e">

**Comment by PowerShellCrack** [comment]
I don't believe Offline media is a supported feature in PSD. **Is it?** When you use the media option in Deployment workbench, it generates the content from the MDT installation's distribution folder, which contains the original MDT VBScripts. This won't work with PSD. 

You could, in theory, copy the contents of the PSD deploymentshare to that folder and build the media from it, but I am not aware of an argument within the PSD modules to use local drive...it will attempt to pull from a http path.

Please correct me if I'm wrong. 

source files
![image](https://github.com/FriendsOfMDT/PSD/assets/20599825/8108c3f0-4a11-4433-a011-0ff25f7e3856)

script folder with vb (Not PSD!
![image](https://github.com/FriendsOfMDT/PSD/assets/20599825/2256a366-873e-445d-a58f-fa8d079ac95f)

**Comment by bretthexum** [comment]
Having offline media would be nice to have.  I can see some scenarios for remote imaging offsite, or providing an MDT image to a vendor to load your image on new devices.  Is this something that could be added as a supported feature?

**Comment by PowerShellCrack** [comment]
i will ping the team and see what can be done.

**Comment by PowerShellCrack** [comment]
@bretthexum in the meantime. I was thinking of alternative options, you could deploy a VM, with the PSD deployment share in place, in one the clouds (I prefer Azure), with a public IP and build a boot ISO from that and give that to vendors to boot from. It does save on sending HDD externals or large DVD's and you won't have to send new content each time...you can just update the deployment shares. Of course, if you update the PSD binaries, then that would need a new ISO or disc.

**Comment by bretthexum** [comment]
> @bretthexum in the meantime. I was thinking of alternative options, you could deploy a VM, with the PSD deployment share in place, in one the clouds (I prefer Azure), with a public IP and build a boot ISO from that and give that to vendors to boot from. It does save on sending HDD externals or large DVD's and you won't have to send new content each time...you can just update the deployment shares. Of course, if you update the PSD binaries, then that would need a new ISO or disc.

Yep, I am actually looking at this solution now.  Hoping we can get Branchecache etc to minimize the WAN traffic
